lib/mixins/readable.js

```diff
@@ -6,7 +6,7 @@ var request = require('../request')
 
 var Readable = {
   get: {
-    value: function (id) {
+    value: function (id, query) {
```

Comment:
In the future it would also be good to add a docstring / type annotation so we know the type of the parameters.

And if we add type annotation, we can also use closure linter for some "static" analysis.